src/utils/valid_input_reg_exp.test.ts

```diff
@@ -0,0 +1,41 @@
+import { expect } from 'chai';
+import isValidInput from './valid_input_reg_exp';
+
+const invalidValues = [
+	'pants',
+	't9.02',
+	'0.2.5',
+	'888.88e80',
+	'..90',
+	'99..',
+	'ultimate.number',
+	'0.00842.2654.433',
+	'R2.D2'
+];
+
+const validValues = [
+	'0',
+	'',
+	'0.',
```

Comment:
Shouldn't this one be an invalid value?

Reply:
No it's a valid entry that a user can make.